.env

```diff
@@ -55,6 +69,15 @@ OKP4_FAUCET_URL='https://faucet.okp4.network/'
 OKP4_SWI_PROLOG_URL='https://www.swi-prolog.org/'
 OKP4_SWI_PROLOG_FUNCTIONS_URL='https://www.swi-prolog.org/pldoc/man?section=functions'
 OKP4_ONTOLOGY_DESCRIPTION_URL='https://blog.okp4.network/okp4-knowledge-sharing-through-ontology-driven-dataverse-a-new-approach-to-knowledge-description-f9b81d4484c8'
+CSV_DATASET_URL='https://data.europa.eu/data/datasets/d043d1f8-f73e-47d1-a471-53156f5e051b?locale=en'
+AWS_S3_URL='https://aws.amazon.com/fr/s3/'
+SCALEWAY_OBJECT_STORAGE_URL='https://www.scaleway.com/fr/object-storage/'
+UUID_IDENTIFIER_URL='https://uuidonline.com/'
```

Comment:
I'd be inclined to promote an opensource project for online `uuid` generation,like [aarreedd/uuidtools](https://github.com/aarreedd/uuidtools.com) - <https://www.uuidtools.com/>. But it's debatable.

Reply:
I provided a random project, you're right. Let use https://github.com/aarreedd/uuidtools.com for `UUID_IDENTIFIER_URL`